lib/mayaUsd/ufe/UsdUndoInsertChildCommand.cpp

```diff
@@ -0,0 +1,161 @@
+//
```

Comment:
This is one of the two core files in the review: the implementation of the UFE insert child command interface.